R/trace_generation.R

```diff
@@ -322,14 +322,18 @@ toBasic <- list(
     })
     group2NA(g, "path")
   },
-  polygon=function(g){
-    if(is.null(g$params$fill)){
-      g
-    }else if(is.na(g$params$fill)){
-      group2NA(g, "path")
-    }else{
-      g
-    }
```

Comment:
geom_polygons are now treated as a basic type so this code is no longer necessary